db/migrate/20181206131627_make_locks_exclusive.foreman_tasks.rb

```diff
@@ -0,0 +1,14 @@
+class MakeLocksExclusive < ActiveRecord::Migration[5.0]
+  def up
+    change_table :foreman_tasks_locks do |t|
+      t.remove :exclusive
+      t.remove :name
+      t.remove_index :name => 'index_foreman_tasks_locks_on_resource_type_and_resource_id'
+      t.index [:resource_type, :resource_id], :unique => true
```

Comment:
Dunno why it's not happening in jenkins, but with postgres on my setup, I'm getting `PG::UniqueViolation: ERROR:  could not create unique index "index_foreman_tasks_locks_on_resource_type_and_resource_id"`. When looking at some production setup, I'm seeing:

```
# in psql
\d foreman_tasks_locks
Indexes:
    "foreman_tasks_locks_pkey" PRIMARY KEY, btree (id)
    "index_foreman_tasks_locks_name_resource_type_resource_id" btree (name, resource_type, resource_id)
    "index_foreman_tasks_locks_on_exclusive" btree (exclusive)
    "index_foreman_tasks_locks_on_name" btree (name)
    "index_foreman_tasks_locks_on_resource_type" btree (resource_type)
    "index_foreman_tasks_locks_on_resource_type_and_resource_id" btree (resource_type, resource_id)
    "index_foreman_tasks_locks_on_task_id" btree (task_id)
```

Reply:
Any `********** NO FILE **********` migrations in the output of `bundle exec rake db:migrate:status`?

Reply:
Oh, I see, it was not due to the conflicting index (I've missed you've deleted the previous on just one line before), but there have still been some duplicates: I need to check whether I haven't run some of the migrations before during testing, that would be skipped now.

Reply:
I'll need to work on the `drop_old_locks` migration a bit more.